tools/ci_build/builds/builds_common.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
```

Comment:
Should we move `ci_testing` into `tools/` as well?`

Reply:
Yeah, how about to move it in next PR?

Reply:
Yeah, probably a good idea until CI testing is finalized